### DISCUSSION TITLE
Copilot Chat access

### DISCUSSION THREAD

**Discussion by pphuppertz** [community/community#106894] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I recently got an email telling me that my organization had granted me access to Copilot Chat, but:
1. it doesn't show up in my marketplace, and:
2. On my github.com/settings/copilot page, Copilot Chat shows up as disabled. 

What am I doing wrong?

**Comment by Akash1134** [comment]
Hi @pphuppertz ,

Have you [followed these steps](https://docs.github.com/en/copilot/managing-copilot-business/managing-access-for-copilot-business-in-your-organization#configuring-access-to-github-copilot-in-your-organization) to configure Copilot and its access on your account?

Thanks!

**Comment by pphuppertz** [comment]
Nope, mainly because the instructions want me to click on "Settings" next to my organization -- but I can only click "Leave" which seems drastic and counterproductive. ;-)

But -- there's something fishy in the state of Denmark: 
In my Copilot settings, Copilot Chat still shows up as Disabled, but in the VS Output for Copilot Chat, I get 
`[Conversations Information] [CopilotClient] Response Content: HasToken: True, ChatEnabled: True, ExpiresAt: 1707510906, RefreshIn: 1500, SKU: copilot_for_business_seat`

**Comment by pphuppertz** [comment, marked as answer]
In other words: it should work now.  And indeed it does.
